src/Controls/src/Core/Platform/GestureManager/GestureManager.Windows.cs

```diff
@@ -481,12 +485,46 @@ void OnPointerReleased(object sender, PointerRoutedEventArgs e)
 			PanComplete(true);
 		}
 
+		void OnPgrPointerEntered(object sender, PointerRoutedEventArgs e) 
+			=> HandlePgrPointerEvent(e, (view, recognizer) 
+				=> recognizer.SendPointerEntered(view, (relativeTo) => GetPosition(relativeTo, e)));
+
+		void OnPgrPointerExited(object sender, PointerRoutedEventArgs e)
+			=> HandlePgrPointerEvent(e, (view, recognizer) 
+				=> recognizer.SendPointerExited(view, (relativeTo) => GetPosition(relativeTo, e)));
+
+		void OnPgrPointerMoved(object sender, PointerRoutedEventArgs e)
+			=> HandlePgrPointerEvent(e, (view, recognizer) 
+				=> recognizer.SendPointerMoved(view, (relativeTo) => GetPosition(relativeTo, e)));
+
+		private void HandlePgrPointerEvent(PointerRoutedEventArgs e, Action<View, PointerGestureRecognizer> SendPointerEvent)
+		{
+			var view = Element as View;
+			if (view == null)
+				return;
+
+			var pointerGestures = view.GestureRecognizers.GetGesturesFor<PointerGestureRecognizer>();
+			foreach (var recognizer in pointerGestures)
+			{
+				SendPointerEvent.Invoke(view, recognizer);
+			}
+		}
+
+		Point? GetPosition(IElement? relativeTo, RoutedEventArgs e)
```

Comment:
Is there a reason why there wouldn't be a Position?

Reply:
We're using Point from Maui.Graphics - to my knowledge, there's no Position there

ccing @PureWeen who implemented GetPosition in his other PR

Reply:
If the user passes an invalid element
The other option here would be to throw an exception